projects/shared/src/lib/form/file-upload/file-upload.component.html

```diff
@@ -1,8 +1,18 @@
-<input
-  #fileUpload
-  type="file"
-  accept="{{ acceptedTypes }}"
-  aria-label="File Upload"
-  [formGroup]="form"
-  [disabled]="form.disabled"
-/>
+<label
```

Comment:
I don't know if this is the correct part of the code to point this out but I tested these changes locally and they look good, however I noticed that the upload file button did not react when I pressed the button, and because it took a few seconds to open the folder with the files, I was unsure if I had pressed the button, making me to press it again
Please make the button react when clicking on it.